apps/bare-expo/ios/Podfile

```diff
@@ -10,6 +10,9 @@ require File.join(File.dirname(`node --print "require.resolve('react-native/pack
 require File.join(File.dirname(`node --print "require.resolve('@react-native-community/cli-platform-ios/package.json')"`), "native_modules")
 require File.join(File.dirname(`node --print "require.resolve('expo-modules-core/package.json')"`), "scripts/autolinking")
 
+require 'json'
+expo_config = JSON.parse(File.read('./expo_config.json')) rescue {}
```

Comment:
Maybe something like `ios.config.json` or `podfile.config.json` so it follows the JS tooling name style. If we want this tied to expo, maybe `ios/.expo/<name>.config.json`

Reply:
Maybe we should also manage this file using a mod so other plugins can safely make changes to it

Reply:
or `podfile.props.json` like `gradle.properties`. (idea from @brentvatne). preferring `podfile.<something>.json` personally and not necessary tied to expo.

Reply:
podfile.properties.json maybe?

Reply:
> podfile.properties.json maybe?

this name looks good to me. going to use this name and if there are any comments, could revise if needed.